### PR TITLE
LPS-137470 Test Autom. Edit goal A/B

### PR DESCRIPTION
**Pull request by mateusralv**
[LPS-97194](https://issues.liferay.com/browse/LPS-97194) 
[LPS-137470](https://issues.liferay.com/browse/LPS-137479)

motivation: As a marketer, I can set the goal of my test on DXP

### CODE REVIEW

**Comment by mateusralv** [comment]
@CarlosBrichete

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/abtest/ABTesting.testcase [hunk 1]
```diff
@@ -108,9 +108,33 @@ definition {
 	@ignore = "true"
```
Comment (CarlosBrichete) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/abtest/ABTesting.testcase:108]: If the test is running, you have to remove this line so the test can be executed in CI.

**Review by CarlosBrichete — CHANGES_REQUESTED** [review]

**Comment by CarlosBrichete** [comment]
Hey @mateusralv , please check the review comment :)

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/abtest/ABTesting.testcase [hunk 2]
```diff
@@ -108,9 +108,33 @@ definition {
...
+			Select(
+			locator1 = "ABTest#CHANGE_SELECT_GOAL",
+			value1 = "Bounce Rate");
+			
+			Click(locator1 = "ABTest#SAVE_AB_TEST_BUTTON");
+
+			Alert.viewSuccessMessage();
+
+			ABTest.runTest();
+		}
```
Comment (CarlosBrichete) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/abtest/ABTesting.testcase]: This test is more simple than that. It is not needed to change the AB test goal. The poshi test goal is to open the Edit modal and check that the goal of the AB test (in this case "Click") is present in the modal. After that, you can just close the modal and terminate the test :)

**Review by CarlosBrichete — CHANGES_REQUESTED** [review]

**Comment by CarlosBrichete** [comment]
Hey @mateusralv , please check the review comment :)

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/abtest/ABTesting.testcase (hunk 1 above)
Comment (mateusralv) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/abtest/ABTesting.testcase:108]: Yes, i forgot about that. I'll fix it.

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/abtest/ABTesting.testcase (hunk 2 above)
Comment (mateusralv) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/abtest/ABTesting.testcase]: Ok, thank you Carlos, I was doubt whether it was necessary or not. i'll fix it too.

**Comment by mateusralv** [comment]
@CarlosBrichete

**Comment by CarlosBrichete** [comment]
ci:test:tango-analytics-cloud

**Comment by CarlosBrichete** [comment]
ci:forward

**Pull request closed without merging**